modules/kubernetes-addons/prometheus/locals.tf

```diff
@@ -1,40 +1,13 @@
 locals {
+  name = "prometheus"
   default_helm_config = {
-    name                       = "prometheus"
-    chart                      = "prometheus"
-    repository                 = "https://prometheus-community.github.io/helm-charts"
-    version                    = "15.3.0"
-    namespace                  = "prometheus"
-    timeout                    = "1200"
-    create_namespace           = false
-    description                = "Prometheus helm Chart deployment configuration"
-    lint                       = false
-    values                     = local.default_helm_values
-    wait                       = true
-    wait_for_jobs              = false
-    verify                     = false
-    set                        = []
-    set_sensitive              = null
-    keyring                    = ""
-    repository_key_file        = ""
-    repository_cert_file       = ""
-    repository_ca_file         = ""
-    repository_username        = ""
-    repository_password        = ""
-    disable_webhooks           = false
-    reuse_values               = false
-    reset_values               = false
-    force_update               = false
-    recreate_pods              = false
-    cleanup_on_fail            = false
-    max_history                = 0
-    atomic                     = false
-    skip_crds                  = false
-    render_subchart_notes      = true
-    disable_openapi_validation = false
-    dependency_update          = false
-    replace                    = false
-    postrender                 = ""
+    name        = local.name
+    chart       = local.name
+    repository  = "https://prometheus-community.github.io/helm-charts"
+    version     = "15.3.0"
+    namespace   = local.name
+    timeout     = "1200"
```

Comment:
do we still need to bump the timeout after resolving the issues caused because of bad dependency setup? ideally prometheus addon should take less than the default 5minute timeout if the overall deployment process is done properly (in the correct order mostly).

Reply:
Lets keep it at 1200 until we get the framework stabilized. We can revisit this later.